File: fabric/src/main/java/gg/norisk/fabric/Fabric.java
package gg.norisk.fabric;

import net.fabricmc.api.DedicatedServerModInitializer;

public class Fabric implements DedicatedServerModInitializer {

  @Override
  public void onInitializeServer() {
    System.out.println("NoRiskClient-Server-API Fabric module is ready!");
  }
} 

File: fabric/src/main/kotlin/gg/norisk/fabric/Fabric.kt
package gg.norisk.fabric

import gg.norisk.fabric.api.NrcChannelRegistrar
import net.fabricmc.api.DedicatedServerModInitializer

class Fabric : DedicatedServerModInitializer {
    override fun onInitializeServer() {
        println("NoRiskClient-Server-API Fabric module is starting...")

        NrcChannelRegistrar.register()

        println("NoRiskClient-Server-API Fabric module is ready!")
    }
}